src/float.rs

```diff
@@ -164,14 +505,54 @@ pub trait FloatCore: Num + NumCast + Neg<Output = Self> + PartialOrd + Copy {
     }
 
     /// Returns `true` if `self` is positive, including `+0.0` and
-    /// `FloatCore::infinity()`.
+    /// `FloatCore::infinity()`, and with newer versions of Rust
+    /// even `FloatCore::nan()`.
```

Comment:
Can we be explicit about the Rust version?

Reply:
Yeah, I'll look that up.

Reply:
Changed the comment to mention that this was Rust 1.20.